### DISCUSSION TITLE
Group with odd elements

### DISCUSSION THREAD

**Discussion by ariefsn** [mjmlio/mjml#2768] [Q&A]
Hi,
So I have 3 elements, it will show as horizontally on large screen.
and the last element will break to next line on small screen.
how to achieve this?
I know can use `mj-group` but seems its will fine with even elements instead of odd.
Please check https://mjml.io/try-it-live/wq7YclCOV1

Thanks

**Comment by iRyusa** [comment]
You need to use the same group for it to work On 24 Oct 2023, at 07:06, Arief Setiyo Nugroho ***@***.***> wrote:﻿
Hi,
So I have 3 elements, it will show as horizontally on large screen.
and the last element will break to next line on small screen.
how to achieve this?
I know can use mj-group but seems its will fine with even elements instead of odd.
Please check https://mjml.io/try-it-live/wq7YclCOV1
Thanks

**Reply by ariefsn** [reply]
@iRyusa thanks for the response.
yea, but the 2nd group width will be 100% because only one element on it.
and the 1st group have 50% each for 2 elements.
also in the 2nd group it will be weird because of center alignment, if set to left not good IMO.
thats why confused.
